launch/process_test.go

```diff
@@ -19,7 +19,8 @@ func TestProcess(t *testing.T) {
 
 // RawCommandValue should be ignored because it is a toml.Primitive that has not been exported.
 var processCmpOpts = []cmp.Option{
-	cmpopts.IgnoreFields(launch.Process{}, "RawCommandValue", "PlatformAPI"),
+	cmpopts.IgnoreFields(launch.Process{}, "PlatformAPI"),
```

Comment:
^^ Looks like the comment above needs to be updated

Reply:
[d3cc6ba](https://github.com/buildpacks/lifecycle/pull/946/commits/d3cc6baa6df958e47430b2f59090fc661fff3f81)